cmd/loadtest/loadtest.go

```diff
@@ -347,32 +351,48 @@ func runLoadTest(ctx context.Context) error {
 		overallTimer = new(time.Timer)
 	}
 
+	// // Dial the Ethereum RPC server.
 	rpc, err := ethrpc.DialContext(ctx, *inputLoadTestParams.RPCUrl)
 	if err != nil {
 		log.Error().Err(err).Msg("Unable to dial rpc")
 		return err
 	}
+	defer rpc.Close()
 	rpc.SetHeader("Accept-Encoding", "identity")
 	ec := ethclient.NewClient(rpc)
 
+	// Define the main loop function.
+	// Make sure to define any logic associated to the load test (initialization, main load test loop
+	// or ven completion steps) in this function in order to handle cancellation signals properly.
 	loopFunc := func() error {
-		err = initializeLoadTestParams(ctx, ec)
-		if err != nil {
+		if initializeLoadTestParams(ctx, ec) != nil {
+			log.Error().Err(err).Msg("Error initializing load test parameters")
 			return err
```

Comment:
maybe i'm wrong, but where does err var get initialized? is it using an err var from earlier?

Reply:
I just forgot to push the [commit](https://github.com/maticnetwork/polygon-cli/pull/147/commits/cf198a93cd9acac605aa8add02e42c2a004ffe23)... 🤦